devlib/utils/misc.py

```diff
@@ -257,6 +257,7 @@ def redirect_streams(stdout, stderr, command):
     def redirect(stream, redirection):
         if stream == subprocess.DEVNULL:
             suffix = '{}/dev/null'.format(redirection)
+            stream = subprocess.PIPE
         elif stream == subprocess.STDOUT:
```

Comment:
We might need to double check the use of redirect_stream in ssh.py as well. Once the pipe is setup, it has to be consumed and closed properly. For Popen uses, it should be ok as Popen's context manager closes the streams on `__exit__()`.

Reply:
Also we might want to do that only for stdout, since stderr should still be DEVNULL as we don't print anything to it.